tests/unit/acquisition/function/test_utils.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2021 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import annotations
+
+import pytest
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+from trieste.acquisition.function.utils import MultivariateNormalCDF
+from trieste.types import TensorType
+
+tfd = tfp.distributions
+
+
+@pytest.mark.parametrize("num_sobol", [-10, -1, 0])
+@pytest.mark.parametrize("dim", [2, 3, 5])
+def test_make_mvn_cdf_raises_exception_for_incorrect_sample_size(
+    num_sobol: int,
+    dim: int,
+) -> None:
+
+    # Set data type and jitter
+    dtype = tf.float64
+
+    with pytest.raises(tf.errors.InvalidArgumentError):
+        MultivariateNormalCDF(sample_size=num_sobol, dim=dim, dtype=dtype)
+
+
+@pytest.mark.parametrize("num_sobol", [1, 10, 100])
+@pytest.mark.parametrize("dim", [-10, -1, 0])
+def test_make_mvn_cdf_raises_exception_for_incorrect_dimension(
+    num_sobol: int,
+    dim: int,
+) -> None:
+
+    # Set data type and jitter
+    dtype = tf.float64
+
+    with pytest.raises(tf.errors.InvalidArgumentError):
+        MultivariateNormalCDF(sample_size=num_sobol, dim=dim, dtype=dtype)
+
+
+@pytest.mark.parametrize("num_sobol", [1, 10, 100])
```

Comment:
looks like a complicated test but maybe we don't care

Reply:
This checks that the batching dimension for the MVNCDF is positive, that is `B > 0`. I don't think it's a complicated test, but it might be unnecessary. However, since I have assumed `B >= 1` in the implementation and throw an error if this is not satisfied, I think it's good to have this test in place (especially since it matches similar conventions assumed in existing tests for acquisition functions). Let me know what you think!

Reply:
Sure I can simplify that!